format-common/src/main/java/io/cdap/plugin/format/plugin/AbstractFileSource.java

```diff
@@ -99,8 +119,18 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) {
   public void prepareRun(BatchSourceContext context) throws Exception {
     FailureCollector collector = context.getFailureCollector();
     config.validate(collector);
-    ValidatingInputFormat validatingInputFormat = context.newPluginInstance(FORMAT_PLUGIN_ID);
     FileFormat fileFormat = config.getFormat();
+    ValidatingInputFormat validatingInputFormat;
+    try {
+      validatingInputFormat = context.newPluginInstance(fileFormat.name().toLowerCase());
+    } catch (InvalidPluginConfigException e) {
+      String errorMessage = String.format("Format '%s' cannot be used because properties %s were not provided or " +
+          "were invalid %s when the pipeline was deployed. Set the format to a " +
```

Comment:
indentation

Reply:
same comment as above about the error message

Reply:
fixed indent and error msg